src/storage/impls.rs

```diff
@@ -1,3 +1,7 @@
+use core::mem::MaybeUninit;
+
+use crate::capacity::{Capacity, MaskingCapacity};
```

Comment:
I don't think these imports are necessary now that we're not modifying the implementations here.